src/functionalTest/java/uk/gov/hmcts/reform/draftstore/features/security/AuthenticatedRequestTest.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.hmcts.reform.draftstore.features.security;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import static com.jayway.restassured.RestAssured.given;
+
+@TestPropertySource("classpath:application.properties")
+@RunWith(SpringRunner.class)
+public class AuthenticatedRequestTest {
+
+    @Value("${test-url}")
+    private String draftStoreUrl;
+
+    @Test
+    public void rejecting_request_when_missing_required_headers() {
```

Comment:
It would be best to stick with the `should...when...` convention. In this case `should_reject_...`

Reply:
This name is for the scenario in testing authenticated request in security scope. This convention comes directly from BDD tests when there are test scenarios for each feature.

Reply:
I would prefer to leave it, but somehow not having strong opinion about it here.